src/validation/coinbase.cairo

```diff
@@ -47,6 +48,10 @@ fn validate_coinbase_input(input: @TxIn, block_height: u32) -> Result<(), ByteAr
         return Result::Err("Previous txid must be zero");
     }
 
+    if *input.previous_output.block_height + COINBASE_MATURITY > block_height {
```

Comment:
That's partially my fault, but coinbase maturity validation happens in `validation::transaction::validate_coinbase_maturity`, you don't have to implement anything, just test it.

We do not validate the coinbase transaction, we validate another transaction that spends a coinbase output from the past.